packages/core/src/components/Tabs/TabList/TabList.tsx

```diff
@@ -108,6 +108,7 @@ const TabList: FC<TabListProps> = forwardRef(
         className={cx(styles.tabsWrapper, className, [getStyle(styles, camelCase(tabType))])}
         id={id}
         data-testid={dataTestId || getTestId(ComponentDefaultTestId.TAB_LIST, id)}
+        data-vibe={ComponentDefaultTestId.TAB_LIST}
```

Comment:
Is it needed? It's part of the Tab
Maybe use a `Tabs` id on the `TabsContext` which is the root of the whole component